### ISSUE TITLE
Layer name from GetLayerName in KiCad 5.1.8

### ISSUE THREAD

**Issue by JLCPCB** [fullyautomated/KiJLC#1]
Hi,

This set of code is neat and compact. I just tested it in KiCad 5.1.8 in Ubuntu 18.04 and find layer name has been changed.

```python
            if layer == "top_copper":
                layer = "top"
                topw.writerow([ref, mid_x, mid_y, layer, rot])
            elif layer == "bottom_copper":
                layer = "bottom"
                botw.writerow([ref, mid_x, mid_y, layer, rot])
```

--
Best Regards,
Atommann
Layout Engineer from JLCPCB

**Comment by tachiniererin** [comment]
I just tried it with a PCB design and can't reproduce any changed layer names. For me it's still the same with KiCad 5.1.8 as it was before. The changes from KiCad 5.1.7 to 5.1.8 also don't seem to change them (https://gitlab.com/kicad/code/kicad/-/commits/5.1.8/).

Can you provide me an example file where you run into this?

**Comment by atommann** [comment]
This morning I tried KiCad 5.1.8 in Ubuntu 18.04 at home and you are right, the plugin has no issue.

```python
            print(layer)
            if layer == "F.Cu":
                layer = "top"
                topw.writerow([ref, mid_x, mid_y, layer, rot])
            elif layer == "B.Cu":
                layer = "bottom"
                botw.writerow([ref, mid_x, mid_y, layer, rot])
```

I add print(layer) in the code and launch KiCad in terminal. In the computer for work, it outputs top_copper and bottom_copper, but today I get F.Cu and B.cu at home. I'll double check on this on Monday.

Edit: I just opened the video project in the demos folder and see what caused my problem: F.Cu was changed to top_copper and B.Cu was changed to bottom_copper :)
